apps/cron/src/notifications/sendGemsPayoutNotifications.ts

```diff
@@ -120,14 +120,14 @@ export async function sendGemsPayoutNotifications({ week }: { week: string }) {
 
   for (const scout of scouts) {
     try {
-      const { points: weeklyClaimablePoints } = await getClaimablePoints({ userId: scout.id, week });
-      if (weeklyClaimablePoints) {
+      const weeklyClaimableTokens = await getClaimableTokens({ userId: scout.id, week });
+      if (weeklyClaimableTokens) {
         await sendNotifications({
           userId: scout.id,
           email: {
             templateVariables: {
               name: scout.displayName,
-              points: weeklyClaimablePoints,
+              points: weeklyClaimableTokens,
```

Comment:
do we need to change the mailgun templates?

Reply:
not necessary since chris already did that